packages/central-server/src/apiV2/projects/EditProject.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { EditHandler } from '../EditHandler';
+import { assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+
+export class EditProject extends EditHandler {
```

Comment:
Also just acknowledging that @acdunham wrote an `EditProject.js` file which is part of the landing pages epic atm, so this'll cause a pretty major conflict when we have to merge those